### PR TITLE
python3Packages.pymsteams: init at 0.1.14

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Python module to interact with  Microsoft Teams

https://github.com/rveachkc/pymsteams

This is a Home Assistant dependency.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 113001` at c7b4064d run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/3caaf0d7642dd8686653a481b444c1eb">python37Packages.pymsteams</a></li></li>
    <li><a href="https://gist.github.com/9e23a6472d78abf757fcd632919fd640">python38Packages.pymsteams</a></li></li>
    <li><a href="https://gist.github.com/4ab93433137e1fcb966e6d771b33c27f">python39Packages.pymsteams</a></li></li>
  </ul>
</details>

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 113001` at c7b4064d run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>3 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/948d40b5f837c987da3a1790293b7130">python37Packages.pymsteams</a></li></li>
    <li><a href="https://gist.github.com/053f89f42e87d6a794ad679ef2523cad">python38Packages.pymsteams</a></li></li>
    <li><a href="https://gist.github.com/a96770daa010c6231d25223c9539ff85">python39Packages.pymsteams</a></li></li>
  </ul>
</details>

File: pkgs/development/python-modules/pymsteams/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  };
+
+  propagatedBuildInputs = [ requests ];
+
+  # Tests require network access
+  doCheck = false;
+  pythonImportsCheck = [ "pymsteams" ];
+
+  meta = with lib; {
+    description = "Python module to interact with  Microsoft Teams";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/pymsteams/default.nix:25]:
```suggestion
    description = "Python module to interact with Microsoft Teams";
```

Comment (fabaff) [reply pkgs/development/python-modules/pymsteams/default.nix]: Thanks, fixed.

**Pull request merged**